scripts/pylint_custom_plugin/dev_requirements.txt

```diff
@@ -0,0 +1,2 @@
+pylint==2.5.2
+astroid>=2.4.1
```

Comment:
I don't think you need the `astroid` pin. `pylint` should bring in the `astroid` dependency at the appropriate version.